src/cli.ts

```diff
@@ -55,12 +60,32 @@ export async function run(): Promise<void> {
 
   if (cliOptions.help) {
     printUsage(argsWithHelp);
+    return;
   }
-  else if (cliOptions.version) {
+  if (cliOptions.version) {
     console.log(getVersion());
+    return;
   }
-  else {
-    await startServer(options);
+
+  const serverInfos = await startServers(options);
+
+  if (serverInfos.length === 1) {
+    const urls = getServerUrls(options, serverInfos[0]!.server);
+    console.log(`Files in this directory are available under the following URLs
+    applications: ${
+                  url.format(urls.serverUrl)}
+    reusable components: ${url.format(urls.componentUrl)}
+  `)
+  } else {
+    const dispatchServer = serverInfos.find(s => s.kind === 'dispatch');
+    if (!dispatchServer) {
+      throw new Error(
```

Comment:
Can this error be pushed down into `startServers`? Seems like this is saying part of it's contract is to start a dispatch server.